test/e2e/storage-endpoint.integration.test.js

```diff
@@ -0,0 +1,165 @@
+const supertest = require("supertest");
```

Comment:
I dont see this file run in the CI

Reply:
e2e test only runs in staging env

Reply:
link or anything ?

Reply:
line number 6 : https://api-ropsten.opencerts.io

Reply:
I mean where do we run this automatically to make sure nothing breaks

Reply:
merge in master. 🗡 breaks means wrong test.